src/orquestra/sdk/_base/cli/_ui/_errors.py

```diff
@@ -176,3 +176,9 @@ def _(e: exceptions.WorkflowRunNotStarted) -> ResponseStatusCode:
 def _(e: exceptions.QERemoved) -> ResponseStatusCode:
     click.echo(e)
     return ResponseStatusCode.CONNECTION_ERROR
+
+
+@pretty_print_exception.register
+def _(e: exceptions.RuntimeQuerySummaryError) -> ResponseStatusCode:
+    click.echo(e)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏We can do better here. Let's spend a few mins making this look nice and actionable for the user.